src/core/jit/builder.rs

```diff
@@ -242,9 +242,37 @@ impl Builder {
         }
     }
 
+    /// Resolves currently processed operation
+    /// based on [spec](https://spec.graphql.org/October2021/#sec-Executing-Requests)
```

Comment:
does this make a real difference in performance?

Reply:
it's a broken logic that doesn't work as intended